balancer-js/src/modules/pools/pool-types/concerns/composableStable/join.concern.ts

```diff
@@ -38,55 +41,73 @@ type SortedInputs = SortedValues &
   };
 
 export class ComposableStablePoolJoin implements JoinConcern {
-  buildJoin = (joinParams: JoinPoolParameters): JoinPoolAttributes => {
-    this.checkInputs(
-      joinParams.tokensIn,
-      joinParams.amountsIn,
-      joinParams.pool.tokensList
-    );
+  buildJoin = ({
+    joiner,
+    pool,
+    tokensIn,
+    amountsIn,
+    slippage,
+    wrappedNativeAsset,
+  }: JoinPoolParameters): JoinPoolAttributes => {
```

Comment:
I destructured `joinParams` so it's following the standard of other concerns. I know it's opinionated, but I feel it's better to keep consistency.